presto-main/src/test/java/com/facebook/presto/operator/scalar/TestUrlFunctions.java

```diff
@@ -87,7 +108,7 @@ public void testUrlDecode()
         }
     }
 
-    private void validateUrlExtract(String url, String protocol, String host, Long port, String path, String query, String fragment)
+    private void validateUrlExtract(String url, String protocol, String host, Long port, String path, String query, String fragment, String rawquery)
```

Comment:
Let's name the new parameter `rawQuery` and place it directly after `query`, so that the two related parameters are together, making it easier to see how the values are different.